### PR TITLE
[RLlib] Fix connector examples

### PR DESCRIPTION
**Pull request by gjoliver**
## Why are these changes needed?

Must have example_X labels for these tests to be actually included.
Also fix data file paths so these example tests can load the checkpoints.
While doing this, also found a bug where we should use ray.cloudpickle to load the checkpoints for now.
Pickle on py37 doesn't support protocol 5 otherwise.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: rllib/BUILD [hunk 1]
```diff
@@ -3343,36 +3343,36 @@ py_test(
...
-    tags = ["team:rllib", "exclusive", "examples", ],
+    tags = ["team:rllib", "exclusive", "examples", "examples_C", "examples_C_AtoT"],
     size = "small",
     srcs = ["examples/connectors/run_connector_policy.py"],
-    data = [
-        "tests/data/checkpoints/APPO_CartPole-v0_checkpoint-6",
-    ],
-    args = ["--checkpoint_file=tests/data/checkpoints/APPO_CartPole-v0_checkpoint-6"]
+    data = glob([
+        "tests/data/checkpoints/APPO_CartPole-v0_checkpoint-6.new",
```
Comment (kouroshHakha) [inline rllib/BUILD:3350]: want to use a better name than .new? :)

File: rllib/utils/policy.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import gym
-import pickle
+import ray.cloudpickle as pickle
```
Comment (kouroshHakha) [inline rllib/utils/policy.py:2]: we have ray.cloudpickle 😮 ?

**Review by kouroshHakha — APPROVED** [review]

File: rllib/utils/policy.py (hunk 2 above)
Comment (gjoliver) [reply rllib/utils/policy.py:2]: yeah, core wants to pin cloudpickle for some reasons.

File: rllib/BUILD (hunk 1 above)
Comment (gjoliver) [reply rllib/BUILD:3350]: yeah, let me change it to date maybe.
these are literally checkpoints from the same cfg, run at different time.

**Pull request merged**